components/alerts/src/js/alert.js

```diff
@@ -0,0 +1,70 @@
+const standardAlert = () => {
+	const error_btn = document.querySelector(".error-btn");
+	const warning_btn = document.querySelector(".warning-btn");
+	const info_btn = document.querySelector(".info-btn");
+	const success_btn = document.querySelector(".success-btn");
+	const error = document.querySelector(".error");
+	const warning = document.querySelector(".warning");
+	const info = document.querySelector(".info");
+	const success = document.querySelector(".success");
+	const rui_cross_success = document.querySelector(".success .rui-cross");
+	const rui_cross_info = document.querySelector(".info .rui-cross");
+	const rui_cross_error = document.querySelector(".error .rui-cross");
+	const rui_cross_warning = document.querySelector(".warning .rui-cross");
+
+	console.log(rui_cross_warning.parentElement);
+	const displayAlert = (element) => {
+		if (error.classList.contains("active")) {
+			error.classList.remove("active");
+			error.classList.add("inactive");
+		}
+		if (warning.classList.contains("active")) {
+			warning.classList.remove("active");
+			warning.classList.add("inactive");
+		}
+		if (info.classList.contains("active")) {
+			info.classList.remove("active");
+			info.classList.add("inactive");
+		}
+		if (success.classList.contains("active")) {
+			success.classList.remove("active");
+			success.classList.add("inactive");
+		}
+		element.classList.remove("inactive");
```

Comment:
```
if (error.classList.contains("active")) {
			error.classList.remove("active");
			error.classList.add("inactive");
		}
``` 
This piece of code is, I feel, similar and repetitive. You can create a separate function here.

Reply:
done